app/views/Project/PillarAnalysis/utils.ts

```diff
@@ -0,0 +1,28 @@
+import {
+    ProjectSourcesQueryVariables,
+} from '#generated/types';
+
+// eslint-disable-next-line import/prefer-default-export
+export function transformSourcesFilterToEntriesFilter(filters: Omit<ProjectSourcesQueryVariables, 'projectId'>) {
```

Comment:
It's best if we do not do these renames.
Renaming fields should also keep in mind renaming all the errors

Reply:
This is no longer required